### PR TITLE
Milestone 3: Query multiple tables

### PR DESCRIPTION
**Pull request by Chrissiku**
### Let's match Screenshots to project requirements 📗 
- Create a table named `owners` with the following columns:
  - `id`: integer (set it as autoincremented PRIMARY KEY)
  - `full_name`: string
  - `age`: integer

1 table owners

- Create a table named `species` with the following columns:
  - `id`: integer (set it as autoincremented PRIMARY KEY)
  - `name`: string

1 table species

- Modify `animals` table:
  - Make sure that id is set as autoincremented PRIMARY KEY

4 set id to be primary key

5 set id to be auto increment on animals tables

  - Remove column `species`

3 edit animals, drop species

  - Add column `species_id` which is a foreign key referencing `species` table
  - Add column `owner_id` which is a foreign key referencing the `owners` table

6 add species_id and owner_id as foreign keys to animals table

- Insert the following data into the `owners` table:
  - Sam Smith 34 years old.
  - Jennifer Orwell 19 years old.
  - Bob 45 years old.
  - Melody Pond 77 years old.
  - Dean Winchester 14 years old.
  - Jodie Whittaker 38 years old.

8 insert into owners

- Insert the following data into the `species` table:
  - Pokemon
  - Digimon

9 insert into species

- Modify your inserted animals so it includes the `species_id` value:
  - If the name ends in "mon" it will be Digimon
  - All other animals are Pokemon

10 Update species for animals tables

- Modify your inserted animals to include owner information (`owner_id`):
  - Sam Smith owns Agumon.
  - Jennifer Orwell owns Gabumon and Pikachu.
  - Bob owns Devimon and Plantmon.
  - Melody Pond owns Charmander, Squirtle, and Blossom.
  - Dean Winchester owns Angemon and Boarmon.

11 Update owners for animals tables

- Write queries (using `JOIN`) to answer the following questions:
  - What animals belong to Melody Pond?
12 Q1

  - List of all animals that are pokemon (their type is Pokemon).
13 Q2

  - List all owners and their animals, remember to include those that don't own any animal.
14 Q3

  - How many animals are there per species?
15 Q4

  - List all Digimon owned by Jennifer Orwell.
16 Q5

  - List all animals owned by Dean Winchester that haven't tried to escape.
17 Q6

  - Who owns the most animals?
18 Q7

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -32,4 +32,143 @@ VALUES
...
+SELECT
+    owners.full_name as owner_full_name,
+    COUNT(animals.name) AS Number_of_animals
+FROM
+    owners
+    LEFT JOIN animals ON owners.id = animals.owner_id
+GROUP BY
+    owner_full_name
+ORDER BY
+    Number_of_animals DESC;
```
Comment (ShadyShawkat) [inline data.sql]: - [x] Kindly make sure that these queries are put in the right file. All queries should be written in the `queries.sql` file.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Chrissiku, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Chrissiku) [reply data.sql]: Thank you @ShadyShawkat. Let me  dive into that right now

**Review by addod19 — APPROVED** [review]
### Hi @Chrissiku, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [x] Great job moving the queries into the right file :100: awesome


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chrissiku** [comment]
Thank you 🤝  @addod19 for taking your time, reading through my project, and approving It.

**Pull request merged**